pkg/api/common/versions.go

```diff
@@ -198,37 +198,99 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.19.0-beta.0":  true,
 }
 
+// AllKubernetesSupportedVersionsOnAzureStack is a whitelist map of all supported Kubernetes version strings on Azure Stack
+// The bool value indicates if creating new clusters with this version is allowed
+var AllKubernetesSupportedVersionsOnAzureStack = map[string]bool{
+	"1.14.7":  false,
+	"1.14.8":  false, // disabled because of https://github.com/Azure/aks-engine/issues/2312
+	"1.15.4":  false,
+	"1.15.5":  false, // disabled because of https://github.com/Azure/aks-engine/issues/2312
+	"1.15.10": false,
+	"1.15.11": true,
+	"1.15.12": true,
+	"1.16.8":  false,
+	"1.16.9":  true,
+	"1.16.10": false,
```

Comment:
`v1.16.10` and `v1.17.6` should be `true`

Reply:
addressed